adoc/admin-troubleshooting.adoc

```diff
@@ -0,0 +1,21 @@
+
+This chapter summarizes frequent problems that can occur while using {productname} and their solutions.
+
+== Overview
+
+This chapter is a collection of frequent problems that are reported for {productname}. Additionally, {suse} support collects problems and their solutions online at link:https://www.suse.com/support/kb/?id=SUSE_CaaS_Platform[] .
+
+In case of problems, a detailed system report can be created with the `supportconfig` command line tool. It will collect information about the system such as: current kernel version, hardware, installed packages, partition setup, and much more. The result is a TAR archive of files. After opening a Service Request (SR), you can upload the TAR archive to Global Technical Support. It will help to locate the issue you reported and to assist you in solving the problem. For details, see link:https://www.suse.com/documentation/sles-12/book_sle_admin/data/cha_adm_support.html[] .
```

Comment:
Should we add a sentence here that there is no sensitive data among this information? "(...) and much more" sounds a little suspicious.

Reply:
Did you confirm there is no sensitive data in what is collected? Since we also collect logs there is likely to be sensitive data in there (user names, error output disclosing sensitive information).

Two good options: 
(Fast) Simply remove the "and so much more" portion since it's colloquial speech anyway and shouldn't be in a document like this.
(Better) Find out what is actually collected and provide a list

I'm fine with either one, if we have the list of stuff collected at hand or can produce it with reasonably low effort then this is preferred.

The phrasing should be changed to something like "and similar information" instead of "much more".

Reply:
I've added a link to the support plugin that lists all the data coillected.